indico/modules/events/management/forms.py

```diff
@@ -270,6 +267,22 @@ def _init_visibility(self, event):
             self.visibility.warning = _("This event's visibility is currently limited by that of '{}'.").format(
                 real_horizon.title)
 
+    def validate_permissions(self, field):
+        entries = [{
+            'principal': principal_from_fossil(principal, allow_emails=True, allow_networks=True),
+            'permissions': permissions
+        } for principal, permissions in field.data]
+        for entry in entries:
+            principal = entry['principal']
+            permissions = entry['permissions']
+            if isinstance(principal, IPNetworkGroup):
+                if set(permissions) - {'_read_access'}:
+                    msg = _('IP networks cannot have management permissions: {}')
+                    raise ValidationError(msg.format(principal.name))
+            if '_full_access' in permissions and len(permissions) != 1:
+                msg = _('"Manage" permission must be unique: {}')
```

Comment:
I remember discussing it but don't remember what we decided in the end... so we always remove explicit permissions once you make someone a full manager?

In any case, assuming this is already done on the client, I don't see a need to create a translatable error message here. You could even just discard these permissions instead of failing.